lib/krb5/fcache.c

```diff
@@ -198,39 +202,56 @@ fcc_lock(krb5_context context, krb5_ccache id,
     return ret;
 }
 
+static krb5_error_code KRB5_CALLCONV
+fcc_get_default_name(krb5_context, char **);
+
+#ifdef WIN32
+#define FILESUBSEP "+"
```

Comment:
Talk to Greg about whether it wouldn't be better to just use the same separator on Unix and Windows...

Reply:
Given Jeff's comment about `:` not being in the safe set in POSIX, I will do so.